pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[build-system]
+requires = ["hatchling>=1.1.0",]
+build-backend = "hatchling.build"
+
+[project]
+name = "luxos"
+version = "0.0.0"
+description = "The all encompassing LuxOS python library."
+readme = "README.md"
+license = { text = "MIT" }  # TODO I don't think this is a MIT??
+requires-python = ">= 3.8"  # TODO we might bump to 3.10
```

Comment:
Full support here, feel free to use a more up to date version.

Reply:
We run tests down to 3.9, but ideally we should pay more attention to 3.11, 3.12 (https://devguide.python.org/versions)